### PR TITLE
Create frontend

### PR DESCRIPTION
**Pull request by KJMoon**
Set up the frontend, with react and webpack, to connect with the backend. Also configured the backend to include nodemon.

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,8 @@ FROM node:18-alpine
 # Create app directory
 WORKDIR /usr/src/app
 
+# RUN npm install --save-dev nodemon
```
Comment (yuzinu) [inline backend/Dockerfile]: We can remove this line

**Review by yuzinu — CHANGES_REQUESTED** [review]
Great job! 🙌 Looks like we can start getting into the nitty gritty!

File: backend/Dockerfile (hunk 1 above)
Comment (KJMoon) [reply backend/Dockerfile]: Removed the line

**Review by yuzinu — APPROVED** [review]
LGTM! 🙌

**Pull request merged**